quantum/split_common/transactions.c

```diff
@@ -750,6 +753,38 @@ static void watchdog_handlers_slave(matrix_row_t master_matrix[], matrix_row_t s
 
 ////////////////////////////////////////////////////
 
+////////////////////////////////////////////////////
+// VELOCIKEY
+
+#if defined(SPLIT_VELOCIKEY_ENABLE)
+
+static bool velocikey_handlers_master(matrix_row_t master_matrix[], matrix_row_t slave_matrix[]) {
+    static uint32_t  last_update = 0;
+    velocikey_sync_t velocikey_sync;
+    velocikey_sync.enabled = velocikey_enabled();
+    velocikey_sync.speed   = velocikey_get_speed();
+    return send_if_data_mismatch(PUT_VELOCIKEY, &last_update, &velocikey_sync, &split_shmem->velocikey, sizeof(velocikey_sync));
+}
+
+static void velocikey_handlers_slave(matrix_row_t master_matrix[], matrix_row_t slave_matrix[]) {
+    velocikey_set_enabled(split_shmem->velocikey.enabled);
```

Comment:
This writes to eeprom each time.  That's probably not a good idea long term.

Reply:
Agreed. Would it make sense to modify `velocikey_set_enabled` to read from eeprom before writing? Or should the current state be rather held in a static variable and only written to eeprom when changed?

Reply:
well, the standard way has been to use a local/static variable, have a `*_noeepron` version of the function that just sets the local variable and the regular function set/read from eeprom.  And then use the noeeprom function for stuff like split syncing.